instrumentation/kamon-spring/src/main/java/kamon/instrumentation/spring/server/CallableWrapper.java

```diff
@@ -0,0 +1,13 @@
+package kamon.instrumentation.spring.server;
+
+import kanela.agent.libs.net.bytebuddy.asm.Advice;
+import java.util.concurrent.Callable;
+
+public class CallableWrapper {
+    @Advice.OnMethodEnter()
+    // raw use of parametrized class?
```

Comment:
I bet you can use `Callable<?>` and it will work out?

Reply:
Worked!